warehouse/query-core/src/test/java/datawave/query/transformer/GroupingTest.java

```diff
@@ -205,28 +205,31 @@ protected BaseQueryResponse runTestQueryWithGrouping(Map<String,Integer> expecte
                         value = Integer.valueOf(fieldBase.getValueString());
                         break;
                     case "GENDER":
-                        genderKey = fieldBase.getValueString();
+                        firstKey = fieldBase.getValueString();
                         break;
                     case "GEN":
-                        genderKey = fieldBase.getValueString();
+                        firstKey = fieldBase.getValueString();
+                        break;
+                    case "BIRTHDAY":
+                        firstKey = fieldBase.getValueString();
                         break;
                     case "AGE":
-                        ageKey = fieldBase.getValueString();
+                        secondKey = fieldBase.getValueString();
                         break;
                     case "AG":
-                        ageKey = fieldBase.getValueString();
+                        secondKey = fieldBase.getValueString();
                         break;
```

Comment:
In theory you could collapse these, but not absolutely required:

```
case "GENDER":
case "GEN":
case "BIRTHDAY":
      firstKey = fieldBase.getValueString()
      break;
case "AGE":
case "AG":
       secondKey = fieldBase.getValueString()
      break;
```


```

Reply:
Got it. I rebased and incorporated your suggestion. Thanks!!